app/views/comfy/admin/api_forms/_form.html.haml

```diff
@@ -92,10 +92,10 @@
               .row.mb-3.align-items-end
                 .form-group.col-12
                   = a.label :prepopulate, 'Prepopulate:', class: 'mr-4'
-                  = a.check_box :prepopulate, checked: properties[prop.label]["prepopulate"] == '1'
+                  = a.check_box :prepopulate, checked: properties[prop.label]&["prepopulate"] == '1'
                 .form-group.col-12
                   = a.label :required, 'Required:', class: 'mr-4'
-                  = a.check_box :required, checked: properties[prop.label]["required"] == '1'
+                  = a.check_box :required, checked: properties[prop.label]&["required"] == '1'
```

Comment:
@donrestarone , I was getting this error because of &
```
undefined method `&' for {"required"=>"1", "placeholder"=>"", "prepopulate"=>"0"}:Hash
```

I have fixed the issue in another PR. https://github.com/restarone/violet_rails/pull/246